docs/meshstack.azure.config.md

```diff
@@ -166,13 +166,18 @@ let example =
 
 ### B2B User Invitation
 
-You can decide if you want Azure to send an automatic email notification about the invitation process to the user by setting `send-azure-invitation-mail` to `true`. Usually this is not needed as meshStack handles
-the invitation notifications.
-
 This configuration is optional. When configured, instructs the replicator to create AAD B2B guest invitations for
-users missing in the AAD tenant managed by this meshPlatform.
+users missing in the AAD tenant managed by this meshPlatform. This configuration is useful if you have one or more
+"workload" AAD Tenants for Azure subscriptions while having a central "home Tenant" for your organization's user
+identities that handles O365 and related services.
+
+Before users can access an AAD Tenant they've been invited to using Azure B2B, they need to go through Azure's
+["Consent Experience"](https://docs.microsoft.com/en-us/azure/active-directory/external-identities/redemption-experience) and accept the invitation. meshStack supports two different entry points into this process:
 
-> B2B Invitations require an email address which is usually fetched from the [euid](./meshstack.identity-federation.md#externally-provisioned-identities).
+- The "Go to Azure Portal" link displayed in meshPanel redirects users into Azure Portal and selects the right AAD tenant and Subscription. This will trigger the Consent Experience in case the user's B2B invitation is pending acceptance.
```

Comment:
```suggestion
- The "Go to Azure Portal" link displayed in meshPanel redirects users into Azure Portal and selects the right AAD tenant and Subscription. This will trigger the consent experience in case the user's B2B invitation is pending acceptance.
```